draft-iab-path-signals-collaboration.md

```diff
@@ -490,29 +490,38 @@ authenticated signal, such as a DNSSEC-authenticated denial of existence
 # Further Work {#research}
 
 This is a developing field, and it is expected that our understanding
-will continue to grow. Among the recent changes are much higher use
-of encryption at different protocol layers and the consolidation of more
-and more traffic to the same destinations; these have greatly
-impacted the field.
+will continue to grow. One recent change is much higher use
+of encryption at different protocol layers. This obviously impacts
+the field greatly, by removing the ability to use most implicit signals.
+But it may also provide new tools for secure collaboration, and force
+a rethinking of how collaboration should be performed.
 
 While there are some examples of modern, well-designed collaboration
-mechanisms, clearly more work is needed. Many complex cases would
+mechanisms, the list of examples is not long. Clearly more work is needed, if
+we wish to realize the potential benefits of collaboration in further cases.
+This requires a mindset change, a migration away from using implicit signals.
+And of course, we need to choose such cases where the collaboration can
+be performed safely, is not a privacy concern, and the incentives of 
+the relevant parties are aligned. Path signals collaboration is not
+for all situations, however. It should also be noted that many complex cases would
```

Comment:
> Path signals collaboration is not for all situations

This sentence seem very generic and I'm not sure I agree.

Reply:
ok, fixing...

Reply:
Yes, thanks!